extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/GZipTest.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.resteasy.reactive;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+import io.vertx.ext.web.Router;
+
+public class GZipTest {
+
+    private static final String APP_PROPS = "" +
+            "quarkus.http.enable-compression=true\n";
```

Comment:
what do you think of a test with the other way?

`quarkus.http.enable-compression = false`

Reply:
If it is **disabled**, should we really perform any sort of test? I don't think that's needed when is not even enabled :)

Reply:
but how to ensure that the property is working correctly?
and how do you make sure that no future code from other developers doesn't affect this and go undetected because you didn't have any tests?

Reply:
@FroMage WDYT

Reply:
Well, we should test the default behaviour, that's true, but because we know what the default is ATM, we want to test the default behaviour without any configuration, to make sure it won't be compressed by default.